ansible/roles_ocp_workloads/ocp4_workload_cee-cf-110/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+become_override: False
+ocp_username: wkulhane-redhat.com
+silent: False
+
+# If your workload needs customization options provide variables to be used.
+# Variable names must be prefixed by the role name "<role name>_<variable>".
+# Therefore because this example workload is called "ocp4_workload_example" the variables
+# should be named "ocp4_workload_example_variable".
+#
+# You can override the defaults as parameters to the Ansible run that runs
+# your workload.
+
+# Documentation for variable 1 (what does it do?)
+#ocp4_workload_example_variable_1: "value 1"
+
+# Documentation for variable 2 (what does it do?)
+#ocp4_workload_example_variable_2: "value 2"
+
+# PnTAE course id should match the name of the grading script, catalog item, and LMS course_code
+ocp4_workload_example_pntae_course_id: "cee-cf-110"
```

Comment:
That can't be quite right, can it? Shouldn't it be `ocp4_workload_cee_cf_110_pntae_course_id` ?